### PR TITLE
changed small_embeddings_prompt function in openai_embeddings.py

### PR DESCRIPTION
**Pull request by Mal-Pat**
Changed the `small_embeddings_prompt` function to calculate the embedding of the `docString` value from `mathlib4-prompts.json` and add the embedding as another value in the dictionary. The dimensions parameter is set to 256.

### CODE REVIEW

**Comment by siddhartha-gadgil** [comment]
Can you edit code instead of commenting out. Easier to git compare

**Comment by siddhartha-gadgil** [comment]
Also do not set the dimension parameter for now. Let it be the default.

File: scripts/openai_embeddings.py [hunk 1]
```diff
@@ -86,3 +75,5 @@ def small_embeddings_descs():
                 count = count + 1
                 print(f"Completed {count}")
             json.dump(out_lines, out, indent=4, ensure_ascii=False)
+
+small_embeddings_prompt()
```
Comment (siddhartha-gadgil) [inline scripts/openai_embeddings.py:79]: Avoid this. Better to open a python console, import and run what we want.

**Comment by Mal-Pat** [comment]
The code has been edited. I think the change in indentation doesn't allow the comparison to be easily seen.

**Review by siddhartha-gadgil — COMMENTED** [review]
Looks good. But we would also want to rerun ada embeddings and run the "small" embeddings for desc.

**Comment by Mal-Pat** [comment]
So for the `mathlib4-descs.jsonl` file, do we write the embeddings for all three fields - `docString`, `description` and `concise-description` in the same dictionary and publish in another file?

**Comment by Mal-Pat** [comment]
I'll change the ada-embeddings function too.

**Comment by siddhartha-gadgil** [comment]
No need to do this for `docString` as the other file has it for many more
definitions and theorems. But yes for the two other fields.

**Comment by Mal-Pat** [comment]
Made all the changes to `ada_embeddings`, `small_embeddings_prompt` and `small_embeddings_descs` in  `openai_embeddings.py`. I ran the first 5 data points from the input files for each of these and it works correctly.

**Comment by siddhartha-gadgil** [comment]
Looks good. Thanks.

**Pull request merged**